tetris.js

```diff
@@ -567,7 +566,7 @@ module.exports = {
 				`);
 				break;
 		}
-		newCoordinates = this.shiftCoordinates(arrayForTetromino, shift);
-		return newCoordinates
+		return newCoordinates = this.shiftCoordinates(arrayForTetromino, shift);
```

Comment:
`return newCoordinates = this.shiftCoordinates(arrayForTetromino, shift);`
is not correct syntax